.gitignore

```diff
@@ -19,3 +19,4 @@ test/docker/tmp
 test/docker/data*
 **/nohup.out
 .tool-versions
+.pre-commit-config.yaml
```

Comment:
this maybe is more useful in your global user gitignore rather than the repo gitignore?